capstone/client/src/components/StoreOverviewCards.js

```diff
@@ -23,11 +23,22 @@ import DriveEtaIcon from '@material-ui/icons/DriveEta';
 import { StoresContext } from './StoresProvider.js';
 import './styles.css';
 
+import safeway from './images/safeway-logo.png';
+import walmart from './images/walmart-logo.jpg';
+import target from './images/target-logo.png';
+
+const logos = {
+  "S" : safeway,
```

Comment:
why don't you just do "safeway": safeway?

Reply:
because safeway (7), walmart (7), and target (6) all have a different amount of letters its easier to just go by the first letter